docs/en/notes/changelog.md

```diff
@@ -2,6 +2,44 @@
 
 ## MMSelfSup
 
+### v1.0.0rc2 (12/10/2022)
+
+The `master` branch is still 0.x version and we will checkout a new `1.x` branch to release 1.x version. The two versions will be maintained simultaneously in the future.
+
+We briefly list the major breaking changes here. Please refer to the [migration guide](../migration.md) for details and migration instructions.
+
+#### Highlight
+
+- Full support of `MAE`, `SimMIM`, `MoCoV3`.
+
+#### New Features
+
+- Full support of `MAE` ([#483](https://github.com/open-mmlab/mmselfsup/pull/483))
+- Full support of `SimMIM` ([#487](https://github.com/open-mmlab/mmselfsup/pull/487))
+- Full support of `MoCoV3` ([#496](https://github.com/open-mmlab/mmselfsup/pull/496))
+
+#### Bug Fixes
+
+- Fix classification configs ([#488](https://github.com/open-mmlab/mmselfsup/pull/488))
+- Fix MAE config name error ([#498](https://github.com/open-mmlab/mmselfsup/pull/498))
+
+#### Improvements
+
+- Refactor colab tutorial ([#470](https://github.com/open-mmlab/mmselfsup/pull/470)))
+- Update readthedocs requirements ([#472](https://github.com/open-mmlab/mmselfsup/pull/472))
+- Update CI ([#476](https://github.com/open-mmlab/mmselfsup/pull/476))
+- Refine mim_slurm_test.sh ([#477](https://github.com/open-mmlab/mmselfsup/pull/477))
```

Comment:
Not just this file, but also other related files. Please check this [PR](https://github.com/open-mmlab/mmselfsup/pull/477)